client/state/ui/README.md

```diff
@@ -2,39 +2,3 @@ UI State
 ========
 
 A module for managing user interface state.
-
-## Actions
```

Comment:
It may be helpful to leave top level descriptions for actions, reducers and selectors in the top level state readme for people unfamiliar with Redux. (No need for the js doc).